tasks/e2e/cypress/integration/05-tutorial-envelop/tutorial-helix-envelop.spec.js

```diff
@@ -0,0 +1,283 @@
+/* eslint-disable no-undef, camelcase */
```

Comment:
Is this essentially just a 1:1 duplication of the current e2e tests, but with envelop set in the `redwood.toml?`

Reply:
If it is then maybe there's a way that we don't have to duplicate this, but can import it from the other tests or something? And just have another way to switch the GraphQL server?

Reply:
> Is this essentially just a 1:1 duplication of the current e2e tests, but with envelop set in the `redwood.toml?`

Not exactly. It:

* Turns on the `useEnvelop` experimental option
* Modifies the `graphql.ts` function to import to
```
export default `import {  createGraphQLHandler,  makeMergedSchema,  makeServices,} from '@redwoodjs/graphql-server'
```

Oh - I guess I could import all the codemod steps that are shared between integration `01-tutorial` and `05-tutorial` w/ envelop flavor. 

Even the spec is the same -- just the 2 codemods for toml and graphql (the first steps) are unique to this integration scenario.

Is that what you mean?

Reply:
Cool - yeah, let's do that then merge, and then work towards getting ol `api` out of the package.

Reply:
I now import the shared codemods for 01 -> 05.

I did try to make the spec a function to share and reuse all the steps, but in the `05-tutorial` it was running both flavors of the tutorial and doubling up.

Since when envelop becomes "standard" we will remove this e2e test in favor or integration tests that use Lambda or Azure or other handlers, I figured one can implement a reusable tutorial that swaps out these handler http cases then.